### ISSUE TITLE
Now workflow calls can be nested

### ISSUE THREAD

**Issue by rhysd** [rhysd/actionlint#201]
https://github.blog/changelog/2022-08-22-github-actions-improvements-to-reusable-workflows-2/

> You can now nest up to 4 levels of reusable workflows giving you greater flexibility and better code reuse.

At least the following workflow should not cause an error.

```yaml
on:
  workflow_call:
    inputs:
      message:
        description: additional message
        required: true
        type: string

jobs:
  call-another-workflow:
    uses: ./.github/workflows/another-workflow.yaml
    with:
      hello: ${{ inputs.message }}
```

actionlint currently reports an error as follows:

```
test.yaml:11:11: reusable workflow cannot be nested. but this workflow hooks "workflow_call" event at line:2,col:3 [workflow-call]
   |
11 |     uses: ./.github/workflows/another-workflow.yaml
   |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by rhysd** [comment]
> 4 levels of reusable

I don't think actionlint can check levels statically (e.g. recursive call using `if:`).

**Comment by rhysd** [comment]
This was fixed in [v1.6.17](https://github.com/rhysd/actionlint/releases/tag/v1.6.17).
